src/app/components/playback/playback_control.rs

```diff
@@ -149,6 +174,41 @@ impl PlaybackControl {
             .expect("error updating icon");
     }
 
+    fn update_shuffle(&self) {
+        let is_shuffled = self.model.state().playback.is_shuffled();
+        let playback_image = if is_shuffled && self.model.is_playing() {
```

Comment:
shouldn't it always be the same image?

Reply:
I have the images change to better reflect if the user has shuffled the playlist or not from the player

Reply:
ah, did not realize this was not a toggle button! i'd prefer it to be a toggle button, with the same icon at all times